### DISCUSSION TITLE
Ability to rotate and re-process files

### DISCUSSION THREAD

**Discussion by eeble** [jonaswinkler/paperless-ng#590] [Feature Requests]
Would it be possible to add an option to rotate and reprocess files?
I have a few that I loaded into my ADF scanner upside-down, and would like an easy way of just saying, "Flip this 180 degrees and reprocess it"

**Comment by jonaswinkler** [comment]
Hello!

Paperless already has the ability to redo OCR on selected documents, see https://paperless-ng.readthedocs.io/en/latest/administration.html#creating-archived-documents.

Concerning rotation: The next version of paperless will have the ability to automatically correct page rotation issues (see #246), and the document archiver management utility will then do that for already existing documents.

**Reply by BlankSlate56** [reply]
I've had this issue as well, having paperless fix the rotation would be great.  Good to hear it is being addressed already.
